src/main/java/uk/gov/hmcts/reform/unspec/model/CaseData.java

```diff
@@ -30,7 +29,6 @@
 
 @Data
 @Builder(toBuilder = true)
-@HasDeemedDateOfServiceTheSameAsOrAfterIssueDate(groups = ConfirmServiceDateGroup.class)
```

Comment:
Can we remove all related code as a part of this ticket? Looks like, annotation, validation group, validator, and tests will be unnecessaary.

Reply:
done